spec/models/conference_room_spec.rb

```diff
@@ -25,9 +25,50 @@
     let(:color) { '#fFB021' }
     let(:conference_room) { build :conference_room, color: color }
 
-    it 'must downcases color' do
+    it 'downcases color' do
       conference_room.valid? # Triggers before_validation
       expect(conference_room.color).to eq color.downcase
     end
   end
+
+  context 'scopes' do
+    let!(:conference_room) { create :conference_room }
+    let!(:conference_room_with_channel) { create :conference_room }
```

Comment:
I'm a bit confused. What do you think about defining `channel` and `epired_channel` before and then inject them to `conference_room_with_channel` and `conference_room_with_expired_channel`?